SDK/AppCenter/Microsoft.AppCenter.Shared/AppCenter.cs

```diff
@@ -11,50 +13,72 @@ namespace Microsoft.AppCenter
     /// </summary>
     public partial class AppCenter
     {
-        // Gets the first instance of an app secret corresponding to the given platform name, or returns the string 
+        const string SecretDelimiter = ";";
```

Comment:
Was this tested on all windows platforms as well to make sure since this is shared code?

Reply:
And both UWP apps (the forms and non forms).

Reply:
yes, it works expected on windows and uwp platforms